lib/rialto/etl/logging.rb

```diff
@@ -10,10 +10,25 @@ module Logging
 
       def logger
         @logger ||= Yell.new do |logger|
-          logger.adapter :datefile, 'rialto_etl.log', level: 'lte.warn' # anything lower or equal to :warn
-          logger.adapter :datefile, 'rialto_etl_error.log', level: 'gte.error' # anything greater or equal to :error
+          logger.adapter :datefile, info_log, level: 'lte.warn' # anything lower or equal to :warn
+          logger.adapter :datefile, error_log, level: 'gte.error' # anything greater or equal to :error
         end
       end
+
+      private
+
+      def log_path
+        return './log/' unless Settings.key?(:log_path)
```

Comment:
I believe you can simplify this entire method to:

```ruby
def log_path
  Settings.log_path || './log/'
end
```

The former will return `nil` if `log_path` is undefined.

Reply:
Thanks! I had this backwards initially and rubocop complained. Brainfart.... thatnks for the commend.